packages/documentation/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -165,7 +175,7 @@ function renderTextarea(args: Args, context: StoryContext) {
       ?disabled=${args.disabled}
       aria-label=${useAriaLabel ? args.label : nothing}
       aria-invalid=${VALIDATION_STATE_MAP[args.validation] ?? nothing}
-      aria-describedby="${args.hint ? 'form-hint-' + context.id : ''}"
+      aria-describedby="${ariaDescribedBy}"
```

Comment:
This writes an empty `aria-describedby=""` when there is no hint or validation message: 
![image](https://github.com/user-attachments/assets/c02daf47-a168-4cd3-bb65-52cf60ae8a45)